### PR TITLE
feat(judicial-system): Add isolation end date

### PR DESCRIPTION
**Pull request by oddsson**
# Isolation end date

https://app.asana.com/0/322488613152836/1199874241701810

## What

Add the ability to enter an isolation end date. Note that

- Isolation can never be set in the past.
- Isolation end date can not be set after the custody end date.

## Why

Isolation should is not always as long as the custody.

## Screenshots / Gifs

Screen Shot 2021-03-12 at 13 23 44
Screen Shot 2021-03-12 at 13 23 28
Screen Shot 2021-03-12 at 13 23 16

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/migrations/20210311090927-update-case.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      queryInterface.addColumn(
+        'case',
+        'isolation_to',
+        {
+          type: 'TIMESTAMP WITH TIME ZONE',
+          defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
```
Comment (gudjong) [inline apps/judicial-system/backend/migrations/20210311090927-update-case.js:11]:
```suggestion
          type: Sequelize.DATE,
```

We use timestamps for 'created' and 'modified' but simple 'date' for other date fields and no default values.

File: apps/judicial-system/backend/test/e2e.spec.ts [hunk 2]
```diff
@@ -322,7 +323,7 @@ function expectCasesToMatch(caseOne: CCase, caseTwo: CCase) {
   expect(caseOne.otherRestrictions || null).toBe(
     caseTwo.otherRestrictions || null,
   )
-
+  expect(caseOne.isolationTo || null).toBe(caseTwo.isolationTo || null)
```
Comment (gudjong) [inline apps/judicial-system/backend/test/e2e.spec.ts:326]: isolationTo must also be handled in caseToCCase the same way other date fields are handled.

**Review by gudjong — CHANGES_REQUESTED** [review]

**Review by gudjong — APPROVED** [review]

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Review by siggakogk — APPROVED** [review]
LGTM 🐶

**Pull request merged**